### PR TITLE
Create Article-on-PHP-loops.md

### PR DESCRIPTION
**Pull request by kimyad812**
Wiki article for PHP Loops.

### CODE REVIEW

**Comment by Rafase282** [comment]
1. Please rename the filename to `PHP-Loops.md`
2. If you want to add code blocks in one line you can use **`** instead of **```**

File: Article-on-PHP-loops.md [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+```
+```
+Output: 
+ x=1 x=2 x=3
+  
+```
+
+### Do...while loop
+
+In ```do...while``` loop the block of code is executed befoe the condition is checked.
```
Comment (Rafase282) [inline Article-on-PHP-loops.md]: before not befoe.

**Comment by Rafase282** [comment]
This look good, ideally we would have more details and examples for each type of loop and also have them in their own article, but we can start with this, thanks @experimentswithtech

**Comment by experimentswithtech** [comment]
i made the changes above mentioned changes thank you

**Comment by Rafase282** [comment]
```diff
-PHP-loops.md
+PHP-Loops.md
```

File: PHP-Loops.md [hunk 2]
```diff
@@ -0,0 +1,131 @@
+# PHP Loops
+
+Loops are blocks of code that execute a specified number of times. Using loops reduces the number of lines in a code.
```
Comment (atjonathan) [inline PHP-Loops.md]: ~~in a code.~~ of code.

File: PHP-Loops.md [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+```
+```
+Output: 
+ x=1 x=2 x=3
+  
+```
+
+## Do...while loop
+
+In `do...while` loop the block of code is executed before the condition is checked.
```
Comment (atjonathan) [inline PHP-Loops.md]: ~~In `do...while` loop~~ In **the** `do...while` loop**,**

**Comment by atjonathan** [comment]
Thanks for your contribution :tada:

**Comment by experimentswithtech** [comment]
thanks :)

**Pull request merged**